examples/dynamo_integration/latency_sensitivity_demo/src/latency_sensitivity_demo/scripts/dynamo_stack.sh

```
@@ -0,0 +1,86 @@
+# Edit the config below, then: ./dynamo-stack.sh
+# Ctrl+C to stop. Logs in /tmp/dynamo-stack/
+#
+# Prerequisites (run once, stays up):
+#   cd dynamo/deploy
+#   docker compose -f docker-compose.yml up -d --remove-orphans
+#   docker compose -f docker-observability.yml up -d --remove-orphans
+set -euo pipefail
+
+# ── Config ───────────────────────────────────────────────────────────────────
+MODEL="nvidia/NVIDIA-Nemotron-3-Nano-30B-A3B-BF16"
+PAGE_SIZE=64
```

Shellcheck (0.11.0)

[warning] PAGE_SIZE appears unused. Verify use (or export if used externally).

(SC2034)